### DISCUSSION TITLE
Scroll zoom in plotly

### DISCUSSION THREAD

**Discussion by Zibc** [Kitware/trame#597] [Q&A] [answered]
Hello,

I was trying and learning how to use the plotly widget.
I am using the following snippet:

```python
fig = go.Figure()
fig.add_trace(go.Scatter(x=[0, 1, 2, 3, 4], y=[0.1, 1, 4, 9, 16], mode="markers", name="Test"))
fig.add_trace(go.Scatter(x=[0, 1, 2, 3, 4], y=[0.5, 5, 9, 13, 60], mode="markers", name="Test"))
fig.update_yaxes(type='log')
plotly.Figure(fig, display_logo=False, display_mode_bar='true', scroll_zoom=True)
```

but the ```scroll_zoom=True``` does not seem to have any effect.
I am not sure whether it's a bug or if I am not using it correctly.

Any hint ?

**Comment by jourdain** [comment, marked as answer]
While what you wrote should have worked, it seems that the plotly component expect an explicit attribute

```python
plotly.Figure(fig, display_logo=False, display_mode_bar=True, scroll_zoom=("true",))
```

**Reply by Zibc** [reply]
Great! Thanks for your quick reply.
Do you see other properties that should be declared in a similar way ?
it doesn't seem to work for all properties ? (this does not seem to work for display_logo for example)

**Reply by jourdain** [reply]
All of them could behave like that since they are passed as wild card attribute. It really depend on how the underneath layer is done. But in that case I'm not sure.
